ibis-server/app/model/__init__.py

```
@@ -64,6 +64,10 @@ class QueryPostgresDTO(QueryDTO):
     connection_info: ConnectionUrl | PostgresConnectionInfo = connection_info_field
 
 
+class QueryRedshiftDTO(QueryDTO):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)